### PR TITLE
refactor: move section "Terminology and conventions" right after "Conformance"

### PR DESCRIPTION
**Pull request by danielpeintner**
resolves part#1 of https://github.com/w3c/wot-scripting-api/issues/342

***
<a href="https://pr-preview.s3.amazonaws.com/danielpeintner/wot-scripting-api/pull/353.html" title="Last updated on Nov 29, 2021, 3:39 PM UTC (132fd6d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-scripting-api/353/ad7a37a...danielpeintner:132fd6d.html" title="Last updated on Nov 29, 2021, 3:39 PM UTC (132fd6d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -362,6 +362,99 @@
...
+    <p>
+      <dfn>WoT network interface</dfn> synonym for <a>WoT Interface</a>
+    </p>
+    <p>
+      <dfn>JSON-LD</dfn> is defined in [[!JSON-LD]] as a JSON document that is augmented with support for Linked Data.
+    </p>
+    <p>
+      <dfn>JSON Schema</dfn> is defined in <a href="https://json-schema.org/specification.html">these specifications</a>.
+    </p>
+    <p>
```
Comment (zolkis) [inline index.html:391]: The terminology up to this point should be fine. But below this should go to the end of the doc, it's more like an addendum. If we use the proper Web platform citations, I think we can remove the stuff from here on.

Comment (zolkis) [reply index.html:391]: More recent specs don't even have this section, as they adhere to the Web Platform and use citations from e.g. https://respec.org/xref/  or bikeshed refs https://tabatkins.github.io/bikeshed/#cli-refs

**Comment by danielpeintner** [comment]
Scripting Call 2021-11-29
* Remove terms and use cross references (comment it out starting from "The terms URL, ..")
* Add Editor Tools Section with XRef link

File: index.html (hunk 1 above)
Comment (danielpeintner) [reply index.html:391]: Note: I linked the tool for ReSpec, see https://github.com/w3c/wot-scripting-api/blob/main/README.md#editors-tools

**Comment by danielpeintner** [comment]
I manged to get rid of all terms except ECMASCRIPT... removing causes lots of issues (defined in several places).

Moreover, [xref](https://respec.org/xref/) does not show proper definitions for "JSON" etc.

Note: I also fixed an issue with `[[value]]` definition in our current version.

**Review by zolkis — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -3940,99 +4037,6 @@ <h3>Denial Of Service Security Risk</h3>
     </section>
   </section>
 
-  <section> <h2>Terminology and conventions</h2>
```
Comment (zolkis) [inline index.html:3943]: Instead of removing, we could just put these in a comment?

Comment (danielpeintner) [reply index.html:3943]: It has not been removed. It was moved to the top as requested.

**Comment by danielpeintner** [comment]
Scripting Call 2021-12-06:
Can be merged

**Pull request merged**